connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -216,6 +218,19 @@ message Expression {
     bool is_user_defined_function = 4;
   }
 
+  message PythonUDF {
```

Comment:
I'm wondering if it makes sense to generalize this message to something like "session UDF" in which there is a oneof for the different languages. 

This will avoid logical duplication for the Scala UDF work (and other languages). 

@hvanhovell  what's your take?

Reply:
Resolved by introducing `ScalarInlineUserDefinedFunction` as suggested.